dev/preview/workflow/preview/deploy-gitpod.sh

```diff
@@ -180,6 +180,23 @@ function installFluentBit {
       upgrade --install fluent-bit fluent/fluent-bit --version 0.21.6 -n "${PREVIEW_NAMESPACE}" -f "$ROOT/.werft/vm/charts/fluentbit/values.yaml"
 }
 
+function installTrustManager {
```

Comment:
How is this change related to this PR?

Reply:
To be able to get a workspace preview running after running the installer

Reply:
Sorry, I don't follow. What in this PR makes it so that we now have to install the trust manager which we didn't need before?